src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnectionOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Net.Quic
+{
+    internal class QuicConnectionOptions
+    {
+        public int BidirectionalStreamCount { get; set; } = 100;
+        public int UnidirectionalStreamCount { get; set; } = 100;
```

Comment:
Can we make these into an accept backlog size? I think this will have better usability than juggling maximums.

Reply:
These essentially are backlog sizes.

Reply:
Oh, excellent. In that case, can we rename these properties to make that more clear?

Reply:
Are they really backlog sizes? I thought these limits applied to the number of concurrent bidirectional and unidirectional streams, not the number that can be in the backlog.

Reply:
I guess I misunderstood exactly what you meant by backlog. These values are the total number of streams that can be open/active (and originate from the peer) at a time.